modules/sdk-coin-hash/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@bitgo/sdk-coin-hash",
+  "version": "1.0.0",
```

Comment:
Will this be ready for publish or should "access": "public" be private until then?